scripts/package/arch-packages/activitywatch-bin/PKGBUILD

```diff
@@ -21,7 +21,7 @@ package() {
     # Symlink executables to /usr/bin
     mkdir -p $pkgdir/usr/bin
     execnames=("aw-server" "aw-watcher-afk" "aw-watcher-window" "aw-qt")
-    for execname in $execnames; do
+    for execname in "${execnames[@]}"; do
```

Comment:
Only seen [*] before, got me googling and fell down a rabbit hole.
According to [SO](https://stackoverflow.com/a/3355375/2127148) the [@] notation seems to be correct in this case.
Will let @ErikBjare take a look at it before merging though